spec/base_spec.rb

```diff
@@ -798,6 +798,7 @@ class DummyReplacedModel < DummyElasticSearchModel; end
             @dummy_model2 = DummyElasticSearchModel.create!(my_string: "Hello2")
             @dummy_model3 = DummyElasticSearchModel.create!(my_string: "Hey1", my_other_string: "Hello3")
             @dummy_model4 = DummyElasticSearchModel.create!(my_string: "Hey2", my_other_string: "Hello4")
+            @dummy_model5 = DummyElasticSearchModel.create!(my_string: "Whatsup")
```

Comment:
Adding the comment here but it pertains to the test file itself:

It looks like we're missing functional test coverage to ensure the `missing` option can be passed to AggregationTerm. We should add a `"with :missing option"` context in `describe ".distinct_values"` and add tests for the three cases
* `missing` for the top level field
* `missing` for the additional fields
* `missing` for the top level field and additional fields

Let me know if you have questions or would like to pair for a bit on this 👍